tf_agents/environments/atari_preprocessing.py

```diff
@@ -224,8 +195,8 @@ def _pool_and_resize(self):
           self.screen_buffer[1],
           out=self.screen_buffer[0])
 
-    transformed_image = cv2.resize(
-        self.screen_buffer[0], (self.screen_size, self.screen_size),
-        interpolation=cv2.INTER_AREA)
-    int_image = np.asarray(transformed_image, dtype=np.uint8)
-    return np.expand_dims(int_image, axis=2)
+    transformed_image = tf.image.resize(
+        images=self.screen_buffer[0][..., np.newaxis],
+        size=(self.screen_size, self.screen_size),
+        method=tf.image.ResizeMethod.AREA)
+    return transformed_image.numpy().astype(np.uint8)
```

Comment:
This would only work in Eager mode, in general we want the core of the library to work both in TF 1.x Graph and TF 2.x Eager.

Reply:
I see, but that's not a regression, since the code was using cv2, which would not work in graph mode, right?

Reply:
I reverted the cv2 change, I'll push a separate PR for that.